api/pom.xml

```diff
@@ -32,7 +32,7 @@
         <!-- test deps -->
         <dependency>
             <groupId>org.hamcrest</groupId>
-            <artifactId>java-hamcrest</artifactId>
```

Comment:
At some point (IIRC), Hamcrest renamed this project and then renamed it back 🤦‍♂